libcxx/utils/generate_feature_test_macro_components.py

```diff
@@ -2296,24 +2374,205 @@ def version_header(self) -> str:
             )
         )
 
+    def header_ftm_data(self, header: str) -> Dict[Std, List[Dict[Ftm, FtmHeaderTest]]]:
```

Comment:
```suggestion
    def test_data(self, header: str) -> Dict[Std, List[Dict[Ftm, FtmHeaderTest]]]:
```

Then you call this like `self.test_data("cstdlib")`, which I guess is reasonably readable?